README.md

```diff
@@ -58,16 +58,30 @@ To look up a document the *docId* is needed. Once a *docId* is known a node can
 
 In Ceramic, each document has a unique identifier (docId). The identifier is a string with the following format:
```

Comment:
Maybe add something about the unique identifier being persistent and never changing

Reply:
Good idea!